providers/dev-guide.md

```diff
@@ -264,6 +264,15 @@ The logic in this class has
 quite strong assumptions on the data being stored. It assumes that it has the tree structure and each entity contains its kids as a nested hash.
 If you are able to achieve that structure in the `refresh_parser.rb`, you are halfway done. Otherwise, good luck :]
 
+### Defining Inventory
+
+Basically, refresh process consists of Collector, Persister and Parser classes (in most providers).
+When Collector loads data from external provider (like AWS), Persister defines structure of Inventory by InventoryCollection objects.  
+Inventory is then saved to VMDB (ManageIQ (also CFME) app database). Each inventory belons to certain ActiveRecord model.
```

Comment:
Maybe just `Inventory is then saved to the database`?

Reply:
> Each inventory belons

typo should be belongs

Reply:
How about `Every inventory collection is mapped to an ActiveRecord model`?

Reply:
I wrote VMDB intentionally because all documentations uses `VMDB` when talking about our database. 
So I rather write both / `VMDB` and `app database`) because at the beginning I was confused what vmdb is. 
I think there is no other place in this doc where VMDB term is explained

Reply:
Also relation between ManageIQ and CFME is explained nowhere (this is not the best place but at least something for new developer)

Reply:
others :+1:

Reply:
> Also relation between ManageIQ and CFME is explained nowhere 

Actually this is intentional, these are upstream docs/guides and should make no mention of CFME